README.md

```diff
@@ -65,7 +68,7 @@ Sync the overlay
 emerge --sync
 ```
 
-#### Ansible playbook
+#### Ansible playbook (Option 2)
 
 The steps described above can be automatically executed by running the Ansible playbook `install.yml` inside the folder `playbooks`. 
 See the `README` in that folder for more details.
```

Comment:
```suggestion
See the `README` in that folder for more details. Note that the playbook will also install a set of packages, e.g. `Lmod` and `archspec`.
```

Reply:
I added one more sentence here, to make it clear that the playbook does a little bit more than the described manual procedure.

Reply:
If those packages do not get installed in the manual installation. Should we add the commands to install them so that one ends in the same scenario?

Reply:
Yes, agreed. Shall I do that in a separate PR? In that case, just ignore the suggestion about adding this sentence.

Reply:
I think we can take advantage of this PR to include it. I can try to have a look to the Ansible playbook and do the steps myself.

Reply:
Okay 👍 
At the moment everything is included in package set `2020.08` (for the current pilot), so it's basically just `emerge @2020.08`, though this will probably change (especially now we're still testing). So, to prevent this from getting outdated very quickly, I'm not sure if we should put the version in the docs. Or maybe just point to [the sets dir in the overlay](https://github.com/EESSI/gentoo-overlay/tree/master/etc/portage/sets) and/or the [defaults file of the playbook](https://github.com/EESSI/compatibility-layer/blob/master/playbooks/roles/compatibility-layer/defaults/main.yml) to find out which sets are available / we installed for the current/latest version of our installation?

Reply:
@bedroge did not see you message and already made a commit.

What do you think about explaining first ansible and then referring to the ansible playblook in the manual installation?

Reply:
I think that would maybe be a good solution, yes. We could say that Ansible is the preferred method for installing the compatibility layer, and then show the manual procedure in the way you suggest with links/pointers to the playbook?